### DISCUSSION TITLE
Shelly EM

### DISCUSSION THREAD

**Discussion by Nazarbayevv** [arendst/Tasmota#16143] [Q&A] [answered]
Hello,

I have Shelly EM, which I want to use for monitoring energy by 2 phases. I find Option129 for split total energy. But I use electric meter with 2 tariffs in my home, I activated Tariff1 and Tariff2 by Tariff<x>, but I can't see splited value for each phase. 
![image](https://user-images.githubusercontent.com/50654913/182576485-6bdda9b3-7701-4f76-9b18-0969b8b9d5a1.png)
Can I use separate value for each phase with splite by Tariff1 and Tariff2?

**Comment by sfromis** [comment, marked as answer]
The [Tariff](https://tasmota.github.io/docs/Commands/#tariff) command does not provide for separate values by meter/phase/channel.

**Reply by Nazarbayevv** [reply]
Thanks. Is any chance to implement this function in the future?
And I tried to reset (or change) EnergyTotal (by EnergyTotal0, EnergyTotal1, EnergyTotal2) and EnergyReset4, but without but without success. I tried:
EnergyTotal0 0,
EnergyTotal1 0.

Where was my wrong?

**Reply by sfromis** [reply]
Since you do not tell what happened (stating "without success" is just a waste of time), or what Tasmota version you are running, no direct answers are possible. The old `EnergyReset` command is no more available since Tasmota version 10, and the replacement commands (EnergyTotal etc) are not available before version 10.

**Reply by Nazarbayevv** [reply]
This is strange situation.
When I use value 0 - any updates, but when I use othe value - it add to my previous value:
Tasmota version is - 12.0.2
![image](https://user-images.githubusercontent.com/50654913/183308831-78ee5642-6a97-4e34-b04a-ee466890cba2.png)

**Reply by Nazarbayevv** [reply]
What I did:
1. EnergyToday1 0; EnergyToday2 0
2. EnergyYesterday1 0; EnergyYesterday2 0
3. After that I could reset EnergyTotal by: EnergyTotal1 0; EnergyTotal2 0.

But how can I reset TotalTariffs?

![image](https://user-images.githubusercontent.com/50654913/183312777-dacee613-5f79-494c-ba3e-7e80f255c312.png)

**Reply by sfromis** [reply]
You did not read the docs for the [EnergyTotal](https://tasmota.github.io/docs/Commands/#energytotal) command? It explains that the value you provide represents midnight, while the output includes the today value, as this is also part of the total.

**Reply by Nazarbayevv** [reply]
Sorry, it is my foul:( Should it correct Tariffs value? Becous I didn't find any info about this.

**Comment by Nazarbayevv** [comment]
I reseted Total consumption, but how can I reset Tariffs value?
![image](https://user-images.githubusercontent.com/50654913/183518391-1808bc9c-dc8c-473a-8daf-1ed64f128dbc.png)

**Reply by arendst** [reply]
Try commands ``EnergyUsage`` and ``EnergyExport``
